src/desktop/apps/artwork/client/index.coffee

```diff
@@ -1,6 +1,7 @@
 { extend, map, compact } = require 'underscore'
 { AUCTION, CLIENT } = require('sharify').data
 { setCookie } = require '../../../components/recently_viewed_artworks/index.coffee'
+{ recordArtworkView } = require 'lib/components/record_artwork_view'
```

Comment:
I moved the component into `lib/` since it can be shared between `desktop` and `mobile`.

And yea, absolute paths seem to be working to more easily require this!

Reply:
Oh nice 👍  for whatever reason I thought this wouldn't work from coffeescript code, but now that I think about it now that we have webpack I think it goes through stages -> coffeescript -> babel -> which then allows us  piggyback on absolute module resolution from our `babelrc`. Could you try to update line 3 above (coffeescript to coffeescript) and see if absolute module resolution works there too? 

Oooh -- Another thing I just through about was tests -- it might break since we're not using webpack in our tests; in that case i think only JS to JS will work between client / server (test) code

Reply:
I was actually struggling a bit to write a meaningful spec here, I just pushed something that wasn't working that I cobbled together...

or did you mean that existing specs would break on these `require` statements? I haven't done a full build, lets see what Circle says.

Reply:
I had to revert this back to a relative path since I couldn't seem to run the (coffeescript) tests locally without it (see https://github.com/artsy/force/pull/2549). Maybe explained by @damassi 's comment above?